services/persistence/main.py

```diff
@@ -1,31 +1,94 @@
 # -*- coding: utf-8 -*-
+import json
 import os
-import manifesto_data
-
-from flask import Flask, jsonify, request
 import sqlite3
 import urllib.parse
 
+from flask import Flask, jsonify, request
+from manifesto_data import ManifestoDataLoader
+
+
 app = Flask(__name__)
 
 DEBUG = True  # os.environ.get('DEBUG') is not None
 VERSION = 0.1
 
+DB_FILENAME = '/db/test.db'
```

Comment:
den pfad koennten wir dann auch aus den env variables nehmen

Reply:
Jep, kann ich auch in der nächsten Iteration ändern